scripting/2.0/modules/ROOT/pages/scripting-module-studio.adoc

```diff
@@ -171,19 +171,18 @@ In the *Configuration XML* editor, place the script in between `<scripting:code>
 You can configure other additional fields for the Execute operation, such as Parameters, Target Variable, and Target Value.
 
 
-=== Parameters
+=== Parameters Parameter
 
-[%header,cols="20s,20a,35a,20a,5a"]
-|===
-| Name | Type | Description | Default Value | Required
-| Parameters | Map<String, Object> | Variables provided to the script as bindings | |
-|===
-
-In the *Parameters* box of the *General* tab, define input parameter values for the script to use through DataWeave. For the DataWeave expression to work correctly, you need to combine these parameters' types output, where keys are strings and values are any object. +
+Use the *Parameters* parameter to define input values for the script to use through DataWeave. For the DataWeave expression to work correctly, you need to combine these parameters' types output, where keys are strings and values are any object. +
 Reference the parameters by name to use them as binding variables, for example: +
 
 `factorial(initialValue + int(payload))`
 
+To configure this parameter for the Scripting Module, follow the next steps:
+
+. Select the name of the connector in the Studio canvas.
+. On the *General* section, add parameter values in the *Parameters* box.
```

Comment:
"on the Studio canvas"

"In the *General* section",

Change "box" to "section"?

Reply:
Both _In_ the Studio Canvas and _on_ the General section, are setences I copied from our Studio template. Are these wrong? 
I'll change parameters box to field, since it's a field box to enter info not a section.